Tools/Postgres/README.md

```diff
@@ -63,3 +63,32 @@ The script below sets the leader to node 2:
 ``` bash
 oc annotate configmaps patroni-postgres-leader leader=patroni-postgres-2 --overwrite=true;
 ```
+
+## Restore Backup
+
+To download the backup to your local computer:
+
+``` bash
+oc rsync <backup-pod-name>:/backups/daily/<date> <local-folder>
+```
+
+Extract backup using gzip:
```

Comment:
What would we use this for?

Reply:
this copies the folder from the pod to the local folder.